tools/get-links.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
Pode deixar umas poucas linhas de comentário explicando o propósito desse script e o uso?

Reply:
Opa, vou colocar uns comentários sim

Reply:
Melhorei o script pra gerar link por caminho também, pode ser necessário em algum caso para forçar instalar as dependências de quem é linkado.